test/onnx/onnx_import_dyn_shapes.in.cpp

```diff
@@ -18,6 +18,8 @@
 #include "ngraph/file_util.hpp"
 #include "ngraph/frontend/onnx_import/default_opset.hpp"
 #include "ngraph/frontend/onnx_import/onnx.hpp"
+#include "nlohmann/json.hpp"
```

Comment:
I'm not sure whether this header landed here deliberately?

Reply:
Yep, it's a leftover from the Resnet50 tests. Thanks for spotting it.